csatar-plugins/knowledgerepository/updates/builder_table_create_csatar_knowledgerepository_trial_systems.php

```diff
@@ -0,0 +1,47 @@
+<?php namespace Csatar\KnowledgeRepository\Updates;
+
+use Schema;
+use October\Rain\Database\Updates\Migration;
+
+class BuilderTableCreateCsatarKnowledgerepositoryTrialSystems extends Migration
+{
+    public function up()
+    {
+        Schema::create('csatar_knowledgerepository_trial_systems', function($table)
+        {
+            $table->engine = 'InnoDB';
+            $table->increments('id')->unsigned();
+            $table->integer('association_id')->unsigned();
+            $table->string('id_string', 15)->nullable();
+            $table->string('name');
+            $table->integer('trial_system_category_id')->nullable()->unsigned();
+            $table->integer('trial_system_topic_id')->nullable()->unsigned();
+            $table->integer('trial_system_sub_topic_id')->nullable()->unsigned();
+            $table->integer('age_group_id')->nullable()->unsigned();
+            $table->integer('trial_system_type_id')->nullable()->unsigned();
+            $table->integer('trial_system_trial_type_id')->nullable()->unsigned();
```

Comment:
Ezekre nem kellene index-et is allitani?

Reply:
A "foreign" miatt kapnak indexet.